docs/models/peering/template.md

```diff
@@ -0,0 +1,26 @@
+# Template
+
+A template is a text that can be parsed using a context processor that will
```

Comment:
Something missing here "A template is a text <something> that can be parsed...". Not sure what the best word is.

Maybe phrase as this:

"Templates are used for rendering device configurations and emails. The template is parsed using a context processor to complete variable references and execute control statements."